Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -163,3 +163,8 @@ spec:
   cephVersion:
     image: quay.io/ceph/ceph:v17.2.5 # Should match external cluster version
 ```
+
+##  Import K8s cluster on existing Rook-Ceph Deployment
+
+1) Copy create-external-cluster-resources.py into /etc/ceph/ directory of toolbox.
```

Comment:
Where do they get this script? How about an example command of copying it?
```suggestion
1) Copy create-external-cluster-resources.py into the directory `/etc/ceph/` of the toolbox.
```

Reply:
how about complete adding a complete path of `deploy/examples/create-external-cluster-resources.py`